<launch>

 <!--   <arg name="enable_imu_trigger" default="false"/>
    <arg name="camera_name" default="hk_camera"/>
    <arg name="camera_frame" default="camera_optical_frame"/>
    
  <node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager" output="screen"/>   -->
  <!-- nodelet'tag should follow obey the following format -->
     <node pkg="nodelet" type="nodelet" name="armor_detect_nodelet" args="load rm_detector/armor_detector nodelet_manager" output="screen" >
     <param name="g_model_path"  type="string" value="$(find rm_detector)/0.66depth640.onnx"/>
     <param name="nodelet_name"  type="string" value="detector_nodelet"/>
     <param name="camera_pub_name"  type="string" value="armor_detector_image_publisher"/>
     <param name="roi_data1_name"  type="string" value="roi_data1"/>
     <param name="roi_data2_name"  type="string" value="roi_data2"/>
     <param name="roi_data3_name"  type="string" value="roi_data3"/>
     <param name="roi_data4_name"  type="string" value="roi_data4"/>
     <param name="roi_data5_name"  type="string" value="roi_data5"/>
<!--      <rosparam command="load" file="$(find rm_detector)/config/camera.yaml" /> -->
     </node>

 <!--   <node pkg="nodelet" type="nodelet" name="hk_camera"
          args="load hk_camera/HKCameraNodelet nodelet_manager"
          output="screen">
        <param name="image_width" value="1440"/>
        <param name="image_height" value="1080"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="$(arg camera_frame)"/>
        <param name="camera_sn" value="00J67375789"/>
        <param name="enable_imu_trigger" value="$(arg enable_imu_trigger)"/>
        <param name="raising_filter_value" value="0"/>
    </node>
-->


</launch>
